.github/CODEOWNERS

```diff
@@ -1,5 +1,4 @@
 # Default owners
-* @Azure/alz-core-team-technical
```

Comment:
Why the removal of this? We did this as part of SFI on most of our repos as it was a recommendation?

Reply:
It is not necessary, it prevents the non-core contributors group from approving changes to the module. E.g. I would like Remo to be able to approve changes, but he cannot due to this line.

Instead this should be governed by write access, which I have implemented using GH teams. Is there a SFI req that says we have to govern 100% of the files with codeowners? We still have codeowners & core team approval required for workflows and codeowner changes.